bottlecap

```diff
@@ -1,104 +0,0 @@
-#!/usr/bin/env bash
-set -euo pipefail
-
-# bottlecap, 'cause it's kinda like cork... get it?
```

Comment:
Ohh.

Reply:
Interestingly, this is actually the last mention of `cork` in the whole tree (excluding vendored sources).